modules/installation-special-config-crony.adoc

```diff
@@ -9,6 +10,10 @@ You can set the time server and related settings used by the chrony time service
 by modifying the contents of the `chrony.conf` file and passing those contents
 to your nodes as a MachineConfig.
 
+.Prerequisites
+* Have a running {product-title} cluster
```

Comment:
The prereqs don't make sense if below you say "If the cluster is not up yet...continue to create the cluster"